core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v1/EventV3Endpoint.java

```diff
@@ -21,35 +21,35 @@
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
 
-@Path("/v3/{organizationId}/events")
+@Path("/v3/{workspaceId}/events")
 @Consumes(MediaType.APPLICATION_JSON)
-@Api(value = "/v3/{organizationId}/events", description = ControllerDescription.EVENT_DESCRIPTION, protocols = "http,https")
+@Api(value = "/v3/{workspaceId}/events", description = ControllerDescription.EVENT_DESCRIPTION, protocols = "http,https")
 public interface EventV3Endpoint {
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = EventOpDescription.GET_BY_TIMESTAMP, produces = ContentType.JSON, notes = Notes.EVENT_NOTES,
-            nickname = "getEventsInOrganization")
-    List<CloudbreakEventsJson> getCloudbreakEventsSince(@PathParam("organizationId") Long organizationId, @QueryParam("since") Long since);
+            nickname = "getEventsInWorkspace")
+    List<CloudbreakEventsJson> getCloudbreakEventsSince(@PathParam("workspaceId") Long workspaceId, @QueryParam("since") Long since);
 
     @GET
     @Path("{name}")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = EventOpDescription.GET_BY_NAME, produces = ContentType.JSON, notes = Notes.EVENT_NOTES,
-            nickname = "getEventsByStackIdInOrganization")
-    List<CloudbreakEventsJson> getCloudbreakEventsByStack(@PathParam("organizationId") Long organizationId, @PathParam("name") String name);
+            nickname = "getEventsByStackIdInWorkspace")
+    List<CloudbreakEventsJson> getCloudbreakEventsByStack(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
 
     @GET
     @Path("struct/{name}")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = EventOpDescription.GET_EVENTS_BY_NAME, produces = ContentType.JSON, notes = Notes.EVENT_NOTES,
-            nickname = " getStructuredEventsInOrganization")
-    StructuredEventContainer getStructuredEvents(@PathParam("organizationId") Long organizationId, @PathParam("name") String name);
+            nickname = " getStructuredEventsInWorkspace")
```

Comment:
Is the space in the nickname intentional?

Reply:
yes